shared/src/designSystem/colors.ts

```diff
@@ -44,6 +44,7 @@ const colors = {
     yvUSDC: "#3E73C4",
     stETH: "#00A3FF",
     AAVE: "#2EBAC6",
+    WAVAX: "#627EEA", // FIXME: Avax
```

Comment:
Get the color hex from Vinnie so we can finalize this

Reply:
Fixed!